### DISCUSSION TITLE
Owl carousel intregration

### DISCUSSION THREAD

**Discussion by deepakjain40** [breezefront/community#183] [3. Integrations]
Hi currently I am working on  my website but I am trying to integrate owl-carousel but not getting integrated and showing error
![image](https://github.com/breezefront/community/assets/141936456/a13138e5-a296-4d9f-aa02-2756cc4ed403)

  on that page  
where data-breeze is implemented I tried to include the cdn link as well then also showing this error 
![image](https://github.com/breezefront/community/assets/141936456/325515f0-35a3-4764-95bf-6868c9082ff6)
can you help me to ingrate me the owl carousel?

**Comment by vovayatsyuk** [comment]
Owl carousel requires jQuery, so it will not work on Breezefront. I recommend using the [built-in breeze slider](https://breezefront.com/docs/slider)
